### PR TITLE
Feature/wizard steps123

### LINKED ISSUE #14: First step (Initial Configuration)
Interface
- [x] DropDownList
- [ ] Link to a Project (Category Option) Checkbox (+ option "not linked to project")
- [x] Show closed project checkbox
- [x] Competencies/teams Panel  (-> multiple selector)

Logic
- [x] Populate competencies/teams panel with data element groupset -> DataElement Group set is Pyj6SCrmnZy but we will retrieve it by the code GL_CoreComp_DEGROUPSET and allow the user to change the code in the settings. Retrieve DataElement Group and display as checkboxs
- [x] Populate checkbox with Projects
- [x] Implement logic to filter by checkbox

### PR DESCRIPTION
**Pull request by tokland**
Fixes #14, #15, #17, #22

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
   "license": "BSD-3-Clause",
```
Comment (adrianq) [inline package.json]: Can we update the author and the license? and maybe set the version to 0.2.0... I am thinking about increasing the second number in every milestone and leave the last one for minor hotfixes in between sprints.

**Comment by adrianq** [comment]
The app is running for me now. I guess you remove the header as it is still failing for you?

**Comment by tokland** [comment]
Yes, it wasn't working. Did you have time to check it? I didn't have time for it.

**Comment by tokland** [comment]
Regarding all your other review comments, you're pretty much right on all of them :) They are minor changes, so I'll update the PR on Tuesday.

**Comment by adrianq** [comment]
Yes, I have tried to fix it as well with no succeed so far. Will keep you posted anyway. No worries about the comments and have a good Easter! :)

**Comment by adrianq** [comment]
Got it!
screenshot from 2017-04-14 21 40 10

It is a bit tricky but as we only need this for prod I think it is more than enough. By the end of this month I am going to Oslo, so I can try to understand how they usually do it.

Basically, I manually copy d2-ui inside dataset-configuration/node-modules and remove dataset-configuration/node-modules/d2-ui/node-modules/d2. Then, I need to babel  dataset-configuration/node-modules/d2-ui and it works. I have uploaded the code in [this branch](https://github.com/EyeSeeTea/dataset-configuration/commits/staticHeader)

**Comment by adrianq** [comment]
Something else to look at, I wrapped up the app as a zip and uploaded the code to a DHIS instance using the app manager. This is the outcome:
screenshot from 2017-04-14 21 57 35

As you can see we are missing some icons... Hopefully it is something silly.

File: package.json (hunk 1 above)
Comment (adrianq) [inline package.json]: Also change icon and name...

**Comment by adrianq** [comment]
Great job @tokland!

**Pull request merged**